src/goto-diff/unified_diff.cpp

```diff
@@ -32,9 +32,8 @@ void unified_difft::get_diff(
 {
   dest.clear();
 
-  differences_mapt::const_iterator entry=
-    differences_map.find(function);
-  if(entry==differences_map.end())
+  differences_mapt::const_iterator entry = differences_map_.find(function);
```

Comment:
Whitespace changes should be in a separated commit. It is hard to see what the actual functional change in this commit is.

Reply:
Done